components/hab/src/config.rs

```diff
@@ -18,26 +18,25 @@ use std::io::Write;
 use std::path::PathBuf;
 
 use hcore::config::{ConfigFile, ParseInto};
+use hcore::env as henv;
 use hcore::fs::{am_i_root, FS_ROOT_PATH};
+use hcore::os::users;
 use toml;
 
 use error::{Error, Result};
 
 const CLI_CONFIG_PATH: &'static str = "hab/etc/cli.toml";
 
 pub fn load() -> Result<Config> {
-    let cli_config_path = cli_config_path();
-    if cli_config_path.exists() {
-        debug!("Loading CLI config from {}", cli_config_path.display());
-        Ok(try!(Config::from_file(&cli_config_path)))
-    } else {
-        debug!("No CLI config found, loading defaults");
-        Ok(Config::default())
-    }
+    common_load(false)
```

Comment:
I'm having a hard time with this abstraction, but I don't have anything better at the moment.  It could be just the function name.  However, since `common-load` is only called by these two functions, it doesn't really matter.